modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -103,7 +103,8 @@ The following procedure describes how to delete SSH connection data of the insta
 These credentials are refreshed every 10 minutes by taskomatic using the defined SSH connection data. Connection to RMT server always uses the last known authentication credentials collected from the {payg} instance.
 
 The status of the {payg} instance credentials collect is shown in the column [literal]``Status`` or on the instance details page.
-When the instance is not reachable, the credential update process will fail.
+When the instance is not reachable, the credential update process will fail and the credentials will become invalid after the second failed refresh.
+Synchronization of Channels will fail when the credentials are invalid. Please keep the connected instances running.
```

Comment:
```suggestion
When the instance is not reachable, the credential update process will fail and the credentials will become invalid after the second failed refresh.
Synchronization of channels will fail when the credentials are invalid.
To avoid this keep the connected instances running.
```

Reply:
not reachable -> unreachable